webapp/src/components/modals/channel_subscriptions/edit_channel_subscription.test.tsx

```diff
@@ -138,7 +138,9 @@ describe('components/EditChannelSubscription', () => {
             <EditChannelSubscription {...props}/>
         );
         wrapper.setState(baseState);
-        wrapper.instance().handleProjectChange('TES');
+        wrapper.instance().handleProjectChange({
```

Comment:
try to add test cases here for the other values of this object.

Reply:
Not required